cmd/abigen/main.go

```diff
@@ -17,124 +17,124 @@
 package main
 
 import (
-    "encoding/json"
```

Comment:
You need to properly configure your editor, or at least run `gofmt -s -w` before submitting

Reply:
my bad, that was an old version